graphql-compiler-plugin-tests/src/test/resources/ballerina_sources/invalid_service_17/service.bal

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/graphql;
+
+type Status record {
+    int capacity;
+    int __elevationgain;
+    boolean night;
+};
+
+type Lift record {
+    string __id;
+    Status getStatus;
+};
+
+service class Trail {
+
+    isolated resource function get id () returns string {
```

Comment:
Not sure if we keep spaces between the accessor name (`id`) and parentheses `()`

Reply:
Then again, it's fine for a test I guess

Reply:
Let's remove the space here.